### DISCUSSION TITLE
Reached the list limit, can't create or edit

### DISCUSSION THREAD

**Discussion by Yonet** [community/community#13463] [Other Feature Feedback, Questions, & Ideas]
Hi,
I loved the lists feature, thank you!
**Issues**:
- Today I've reached my limit of 16 and can't create new ones or edit the existing ones to consolidate. Ex: I have JS & TS list that I can make one list of. 
- List limit notification is only after I create the + icon and type in the list name and click create again. Expected it the "+ Create list" drop down to be disabled instead. See image below.
<img width="350" alt="githubListsFB" src="https://user-images.githubusercontent.com/3523671/160030594-3d30e289-b455-4636-b4b0-e8a996ff59d0.png">
- I also don't know where to find my lists. 

**Suggestions:**
- Infinite list options 🤓
- An intuitive UI to view, edit and share the created lists.

Thanks!

**Comment by cliffordfajardo** [comment]
Also reached 16 really quick;, I think an upper bound limit like 100 would keep us happy for a while 😄 

<img src="https://user-images.githubusercontent.com/6743796/160181648-ccda2b18-1423-4163-9126-1be922f3972b.png" height="300"/>


Related
- https://github.com/github/feedback/discussions/8608
- https://github.com/github/feedback/discussions/8633
- https://github.com/github/feedback/discussions/9672#discussion-3781923

**Reply by andrei0x309** [reply]
Yeah, 100 at minimum, the whole concept of making list is to group things in niches and for that, you need many lists/categories.

 Also, I don't know but maybe this was mentioned ( I'll search to see) but a reorder feature would be great too.

**Reply by jsoref** [reply]
This is covered in: https://github.com/github/feedback/discussions/8633

**Reply by Yonet** [reply]
Thank you 😊

**Comment by loretoparisi** [comment]
+1 for more then 16, possible options could be

25
50
100

And / or 

- Edit
- Remove
